ezomero/_posts.py

```diff
@@ -237,7 +246,7 @@ def post_map_annotation(conn, object_type, object_id, kv_dict, ns,
     >>> post_map_annotation(conn, "Image", 56, d, ns)
     234
     """
-
+    print("group at begin", conn.getGroupFromContext().getId())
```

Comment:
What is the significance of this line, and 276?

Reply:
that was me trying to debug the across-groups issue and forgetting to remove prints :)